release_notes.md

```diff
@@ -3,4 +3,5 @@
 - My change description (#PR/#issue)
 -->
 
-- Implementation for bypass deferred binding (#1462/#1495)
\ No newline at end of file
+- Implementation for bypass deferred binding (#1462/#1495)
+- Support binding to ServiceBusReceivedMessage (#1313)
```

Comment:
Should be using the service bus release notes file: https://github.com/Azure/azure-functions-dotnet-worker/blob/main/extensions/Worker.Extensions.ServiceBus/release_notes.md